roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
I can see Jamie's point of view too. Mostly I want to avoid this data being defined solely in inventory files. We have to either be clever with some default inventory variables (a shared "all" group), implement a plugin to express default values for all inventories (like Ursula has), or remember to copy/paste/update the data into every inventory.

Perhaps we can have some roles that are meant to be shared, like zuul, and some roles that are our implementation specific, like the log rotate role. The data for logs to rotate may have to be duplicated, as some of the info may exist in the zuul role, but if we don't want to establish a relationship we'll have to duplicate it.